.github/ISSUE_TEMPLATE/new_component_template.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Nieuw component
+about: Nieuw component in LUX Design System
```

Comment:
Het doel van `about` is mij niet duidelijk. Graag een passende placeholder plaatsen